mixer/adapter/list/list_integration_test.go

```diff
@@ -98,3 +140,102 @@ func TestReport(t *testing.T) {
 		},
 	)
 }
+
+func TestReportBlacklist(t *testing.T) {
+	adapter_integration.RunTest(
+		t,
+		GetInfo,
+		adapter_integration.Scenario{
+			ParallelCalls: []adapter_integration.Call{
+				{
+					CallKind: adapter_integration.CHECK,
+					Attrs:    map[string]interface{}{"source.name": "src1"},
+				},
+				{
+					CallKind: adapter_integration.CHECK,
+				},
+			},
+			Configs: []string{
+				h1BlacklistOverrideSrc1Src2,
+				r1H1I1,
+				i1ValSrcNameAttr,
+			},
+			Want: `{
+            "AdapterState": null,
+            "Returns": [
+             {
+              "Check": {
+                "Status": {
+                "code": 7,
+                "message": "staticversion.listchecker.istio-system:src1 is blacklisted"
+               },
+               "ValidDuration": 300000000000,
+               "ValidUseCount": 10000
+              }
+             },
+             {
+              "Check": {
+                "Status": {},
+                "ValidDuration": 300000000000,
+                "ValidUseCount": 10000
+              }
+             }
+            ]
+            }`,
+		},
+	)
+}
+
+func TestReportIpWhitelist(t *testing.T) {
+	adapter_integration.RunTest(
+		t,
+		GetInfo,
+		adapter_integration.Scenario{
+			ParallelCalls: []adapter_integration.Call{
+				{
+					CallKind: adapter_integration.CHECK,
+					Attrs: map[string]interface{}{"source.ip": "10.56.0.1",
```

Comment:
IPs are represented as `[]byte` in attributes.

Reply:
Thanks for pointing... will update